game.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+class Game < Item
+  attr_accessor :multiplayer, :last_played_at, :title
+
+  def initialize(title, publish_date, multiplayer, last_played_at, genre, label)
+    super(publish_date)
+    @title = title
+    @multiplayer = multiplayer
+    @last_played_at = last_played_at
+    @genre = genre
+    @label = label
+  end
+
+  def can_be_archived?
+    @last_played_at < 2.year.ago
+  end
```

Comment:
- [ ] Similar to the Item class method, you will need to use the date library to verify how older is the game, please check the comment above. 👆🏽